kogitoq/gadaq/src/main/java/life/genny/gadaq/live/data/InternalConsumer.java

```diff
@@ -127,11 +131,54 @@ public void getEvent(String event) {
 
 		log.info("Received Event : " + SecurityUtils.obfuscate(event));
 
+		// Quick search
+		if(filter.isQuickSearchDropdown(msg)) {
+			filter.sendListQuickSearchDropdown(msg);
+			return;
+		}
+
 		// If the event is a Dropdown then leave it for DropKick
 		if ("DD".equals(msg.getEvent_type())) {
 			return;
 		}
 		events.route(msg);
+
+		// Saved search buttons
+		if(filter.isFilterBtn(msg)) {
+			filter.handleBtnEvents(msg);
+		}
+
+		scope.destroy();
+		Instant end = Instant.now();
+		log.info("Duration = " + Duration.between(start, end).toMillis() + "ms");
+	}
+
+	/**
+	 * Consume data from anwsers
+	 * @param event Event
+	 */
+	@Incoming("data")
```

Comment:
We should not be consuming from the `data` topic here. We are already consuming from the `valid_data` topic above.

Reply:
valid_data is not working for dropdown